s3/replication/replicator/src/s3replicator/replicator_routes.py

```diff
@@ -107,6 +109,8 @@ async def abort_job(request):
     # XXX Perform real abort...
     job = request.app['all_jobs'].remove_job_by_job_id(job_id)
     if job is not None:
+        # Perform the abort
+        job.abort()
         _logger.debug('Aborted Job with job_id {}'.format(job_id))
         return web.json_response({'job_id': job_id}, status=204)
```

Comment:
TODO : With 204 status we should not send any kind of body as a part of response. This change can be taken in next PRs.